stock_available_to_promise_release/views/stock_picking_views.xml

```diff
@@ -7,15 +7,125 @@
+        <field name="model">stock.picking</field>
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="stock.vpicktree" />
+        <field eval="900" name="priority" />
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="zip_code" optional="hide" />
+                <field name="city" optional="hide" />
+                <field name="state_id" options="{'no_open': True}" optional="hide" />
+            </field>
+            <field name="origin" position="after">
+                <field name="group_id" optional="hide" />
+            </field>
+            <tree position="inside">
+                <field name="date_priority" optional="show" />
+                <field name="need_release" invisible="1" />
+                <field name="release_ready" />
+                <field name="release_ready_count" />
+                <button
+                    name="action_open_move_need_release"
+                    string="View Move Allocations"
+                    title="View Move Allocations"
+                    icon="fa-cubes"
+                    type="object"
+                    attrs="{'invisible': [('need_release', '=', False)]}"
+                />
+            </tree>
+        </field>
+    </record>
+    <record id="view_picking_release_search" model="ir.ui.view">
+        <field name="name">stock.picking.release.search</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_internal_search" />
+        <field name="arch" type="xml">
+            <filter name="late" position="before">
+                <filter
+                    string="Release Ready"
+                    name="release_ready"
+                    domain="[('release_ready', '=', True)]"
+                />
+                <separator />
+            </filter>
+            <filter name="late" position="attributes">
+                <attribute
+                    name="domain"
+                >[('scheduled_date','&lt;', time.strftime('%%Y-%%m-%%d %%H:%%M:%%S')), ('state', 'in', ('assigned', 'waiting', 'confirmed')), ('need_release', '=', False)]</attribute>
+            </filter>
+            <filter name="waiting" position="attributes">
+                <attribute name="domain">
+                    [('state', 'in', ('confirmed', 'waiting')), ('need_release', '=', False)]
+                </attribute>
+            </filter>
+        </field>
+    </record>
+    <record id="stock_picking_release_action" model="ir.actions.act_window">
+        <field name="name">Transfers Allocations</field>
+        <field name="res_model">stock.picking</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="view_mode">tree,kanban,form</field>
+        <field name="search_view_id" ref="stock.view_picking_internal_search" />
+        <field
+            name="view_ids"
+            eval="[(5, 0, 0),
```

Comment:
why this 1st line?

Reply:
It resets the m2m, this is what's done in every core addon where `view_ids` is set. The alternative would be to use a `[(6, 0, ...)]` but I prefer consistency.